documents/markdown/guides/getting-started.md

```diff
@@ -47,14 +47,15 @@ platform is highly flexible, configurable, and adaptable.
 The following terminology will help you understand the concepts around
 which OPP is designed:
 
-+ Hub Key &mdash;
-+ Asset &mdash;
-+ Offer &mdash;
-+ Agreement &mdash;
-+ Licensor &mdash;
-+ Open Licensing &mdash;
-
-Because both OPP and the Open Permision ODRL extension are open source, you
++ Hub Key &mdash; a unique resource locator generated by the Open
+  Permissions Platform to locate an entity
++ Asset &mdash; an entity having potentially licensable content
++ Offer &mdash; terms of use offered for an entity
++ Agreement &mdash; an offer agreed between a licensor and user
++ Licensor &mdash; a party that has the right to offer assets
++ Open Licensing &mdash; the ODRL profile used by the Open Permissions Platform
```

Comment:
No such thing as Open Licensing any more.
Any references to it should be removed, the name of ODRL profile we are using is called Open Permissions also.